src/closures.rs

```diff
@@ -238,9 +238,21 @@ fn rewrite_closure_fn_decl(
         .shrink_left(is_async.len() + mover.len() + immovable.len())?
         .sub_width(4)?;
 
+    let indent_style = context.config.indent_style();
+    let version = context.config.version();
+
     // 1 = |
     let param_offset = nested_shape.indent + 1;
-    let param_shape = nested_shape.offset_left(1)?.visual_indent(0);
+    let param_shape = match indent_style {
+        IndentStyle::Block => {
+            if version == Version::Two {
```

Comment:
I should drop the version gating from here since this would be a 2.x+ change correct?

Reply:
Yes, I will remove the version anyway so it would be helpful if you can remove it in this PR :)